cloud/amazon/public/resources/subnet.go

```diff
@@ -247,7 +248,12 @@ func (r *Subnet) tag(tags map[string]string) error {
 			Value: S("%s", val),
 		})
 	}
-	_, err := Sdk.Ec2.CreateTags(tagInput)
+	req, _ := Sdk.Ec2.CreateTagsRequest(tagInput)
+	retry := true
+	req.Retryable = &retry
+	req.RetryCount = 30
```

Comment:
Ideally these would go up as constants

Reply:
If you want to update feel free - otherwise I will just merge - LMK what you think